<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LibreCellular RFE Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>131</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string> Comms Port </string>
    </property>
    <widget class="QPushButton" name="btnOpenClosePort">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Port</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboPorts">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>561</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string> Power </string>
    </property>
    <widget class="QPushButton" name="btnPort1On">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPort2On">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPort1Off">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPort2Off">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ext. Port 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ext. Port 2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnUpdateVolts">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Update Voltages</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string> 5V </string>
     </property>
     <widget class="QLabel" name="lbl5V">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>? V</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string> 12V </string>
     </property>
     <widget class="QLabel" name="lbl12V">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>? V</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>20</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string> 24V </string>
     </property>
     <widget class="QLabel" name="lbl24V">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>? V</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn12VOn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn12VOff">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>12V Buck</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn5VOn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>5V Buck</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn5VOff">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>861</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string> RF Operations </string>
    </property>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>401</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string> Channel A </string>
     </property>
     <widget class="QPushButton" name="btnLNA_A_Active">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Active</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>LNA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLNA_A_Bypass">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Bypass</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>PA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPA_A_Active">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Active</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPA_A_Bypass">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Bypass</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>170</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>RX Attenuation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSetAtten_A">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>190</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboRXAtten_A">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string> Power Level </string>
      </property>
      <widget class="QLabel" name="lblPowerLevel_A">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnPwrLevel_A_Read">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Power Measurement</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPwrMeas_A_Set">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboPwrMeasMode_A">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TX/RX Loopback</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTXRXLoop_A_Inactive">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>190</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Inactive</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTXRXLoop_A_Active">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Active</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnReset_A">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset Channel A Operation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTXInhibit_A_Active">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Active</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTXInhibit_A_Inactive">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Inactive</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TX Inhibit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>190</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>dB</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>401</width>
       <height>291</height>
      </rect>
     </property>
     <property name="title">
      <string> Channel B </string>
     </property>
     <widget class="QPushButton" name="btnLNA_B_Active">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Active</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>LNA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLNA_B_Bypass">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Bypass</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>PA</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPA_B_Active">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Active</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPA_B_Bypass">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Bypass</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>170</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>RX Attenuation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSetAtten_B">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>190</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboRXAtten_B">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string> Power Level </string>
      </property>
      <widget class="QLabel" name="lblPowerLevel_B">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnPwrLevel_B_Read">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>100</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Read</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Power Measurement</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPwrMeas_B_Set">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboPwrMeasMode_B">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TX/RX Loopback</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTXRXLoop_B_Inactive">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>190</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Inactive</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTXRXLoop_B_Active">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Active</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnReset_B">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset Channel B Operation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTXInhibit_B_Active">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Active</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTXInhibit_B_Inactive">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Inactive</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TX Inhibit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>190</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>dB</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="lblLogo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>(logo)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
